apps/studio.giselles.ai/app/sentry-example-page/page.tsx

```
@@ -0,0 +1,200 @@
+          <span>
+            Throw Sample Error
+          </span>
+        </button>
+
+        {hasSentError ? (
+          <p className="success">
+            Sample error was sent to Sentry.
+          </p>
+        ) : !isConnected ? (
+          <div className="connectivity-error">
+            <p>The Sentry SDK is not able to reach Sentry right now - this may be due to an adblocker. For more information, see <a target="_blank" href="https://docs.sentry.io/platforms/javascript/guides/nextjs/troubleshooting/#the-sdk-is-not-sending-any-data">the troubleshooting guide</a>.</p>
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)